libsndfile will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libsndfile it will fail in the following way:

```
2024/11/12 23:12:26 INFO running step "test/pkgconf"
2024/11/12 23:12:26 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:26 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:26 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:26 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:27 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + basename /home/build/melange-out/libsndfile-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + dev_pkg=libsndfile-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + apk info -L libsndfile-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + grep -q ^Name: usr/lib/pkgconfig/sndfile.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + basename usr/lib/pkgconfig/sndfile.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + lib_name=sndfile uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + echo usr/lib/pkgconfig/sndfile.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + grep -q '^usr/lib/pkgconfig/sndfile.pc$\|^usr/share/pkgconfig/sndfile.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + pkgconf --exists sndfile uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + grep -q ^Version: usr/lib/pkgconfig/sndfile.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + pkgconf --modversion sndfile uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 INFO 1.2.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + grep -q ^Libs: usr/lib/pkgconfig/sndfile.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + pkgconf --libs sndfile uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 INFO -lsndfile uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + grep -q ^Cflags: usr/lib/pkgconfig/sndfile.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN + pkgconf --cflags sndfile uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN Package flac was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN Perhaps you should add the directory containing `flac.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN Package 'flac', required by 'sndfile', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN Package 'ogg', required by 'sndfile', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN Package 'vorbis', required by 'sndfile', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN Package 'vorbisenc', required by 'sndfile', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 WARN Package 'opus', required by 'sndfile', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:27 INFO pod b3797e84bfb974b12f72c8370f3562ead3a40bf38e967d2e7eb724cd208a1f8d terminated
2024/11/12 23:12:27 INFO pod 1285909949af69e6514fc24d3345623d804e5f14196b39a7cb4f230795d63de2 terminated
2024/11/12 23:12:27 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:27 INFO   workspace dir: /home/user/tmp/melange-workspace-2256253007
2024/11/12 23:12:27 INFO   guest dir: /home/user/tmp/melange-guest-4267570291
2024/11/12 23:12:27 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libsndfile] Error 1
```

--- a/libsndfile.yaml
+++ b/libsndfile.yaml
@@ -2,7 +2,7 @@
 package:
   name: libsndfile
   version: 1.2.0
-  epoch: 2
+  epoch: 3
   description: C library for reading and writing files containing sampled sound
   copyright:
     - license: LGPL-2.1-or-later
@@ -67,7 +67,15 @@ subpackages:
   - name: libsndfile-dev
     pipeline:
       - uses: split/dev
+    dependencies:
+      runtime:
+        - libogg-dev
+        - libvorbis-dev
+        - opus-dev
     description: libsndfile dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libsndfile-doc
     pipeline:

